ISICbalance/ViewControllers/BalanceViewController.swift

```diff
@@ -0,0 +1,122 @@
+//
+//  BalanceViewController.swift
+//  ISICbalance
+//
+//  Created by Rostislav Babáček on 12/03/2019.
+//  Copyright © 2019 Rostislav Babáček. All rights reserved.
+//
+
+import UIKit
+import SwiftKeychainWrapper
+import SnapKit
+
+class BalanceViewController: BaseViewController {
+    private var viewModel: BalanceViewModel
+    
+    weak var screenStackView: UIStackView!
+    weak var balanceLabel: UILabel!
+    weak var balanceTitle: UILabel!
+    weak var reloadButton: UIButton!
+    weak var accountButton: UIButton!
+    
+    override init() {
+        self.viewModel = BalanceViewModel()
+        super.init()
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    override func loadView() {
+        super.loadView()
+        self.view.backgroundColor = UIColor.Theme.backgroundColor
+        
+        let screenStackView = UIStackView()
+        screenStackView.axis = .vertical
+        screenStackView.spacing = 60
+        self.screenStackView = screenStackView
+        self.view.addSubview(screenStackView)
+        
+        screenStackView.snp.makeConstraints { make in
+            make.centerX.centerY.equalToSuperview()
+            make.leading.greaterThanOrEqualTo(20)
+            make.trailing.lessThanOrEqualTo(-20)
+        }
+        
+        setupBalanceField()
+        setupButtonsStack()
+    }
+    
+    fileprivate func setupBalanceField() {
+        let balanceTitle = UILabel()
+        balanceTitle.text = L10n.Balance.title
+        balanceTitle.textColor = UIColor.Theme.labelBlue
+        balanceTitle.textAlignment = .center
+        balanceTitle.font = UIFont.systemFont(ofSize: 18)
+        self.balanceTitle = balanceTitle
+        screenStackView.addArrangedSubview(balanceTitle)
+        
+        let balanceLabel = UILabel()
+        #warning("TODO")
+        balanceLabel.text = "1357 Kč"
+        balanceLabel.textColor = UIColor.Theme.textColor
+        balanceLabel.adjustsFontSizeToFitWidth = true
+        balanceLabel.textAlignment = .center
+        balanceLabel.font = UIFont.boldSystemFont(ofSize: 80)
+        self.balanceLabel = balanceLabel
+        screenStackView.addArrangedSubview(balanceLabel)
+    }
+    
+    fileprivate func setupButtonsStack() {
+        let buttonsStackView = UIStackView()
+        buttonsStackView.spacing = 100
+        screenStackView.addArrangedSubview(buttonsStackView)
+        
+        let reloadButton = UIButton()
+        reloadButton.setImage(UIImage(asset: Asset.reloadIcon), for: .normal)
+        self.reloadButton = reloadButton
+        buttonsStackView.addArrangedSubview(reloadButton)
+        
+        let spacerView = UIView()
+        buttonsStackView.addArrangedSubview(spacerView)
+        
+        let accountButton = UIButton()
+        accountButton.setImage(UIImage(asset: Asset.accountIcon), for: .normal)
```

Comment:
používáš-li Swiftgen, tak lepší zápis je `accountButton.setImage(Asset.accountIcon.image, for: .normal)`, je to kratší, šetři si prsty 😄